enhancements/cluster-logging/hypershift-audit-logs.md

```diff
@@ -21,174 +21,124 @@ see-also:
 
 **NOTE**: This proposal depends on [api-audit-log-policy](api-audit-log-policy.md)
 Please read that proposal for details not covered here.
 
 ## Motivation
 
+- Forward API server audit logs from the management cluster to the hosted cluster, based on hosted cluster configuration.
+- Allow hosted cluster user to forward HCP audit logs in the same way as logs collected from the hosted cluster.
 - Allow audit logs consumers with different requirements to get the subset logs they need.
-- Forward API server audit logs from the management cluster, as they are not available in the hosted cluster.
 - Replace the SD Splunk forwarder with the CLO which is more flexible and feature-rich.
-- Separate configuration so that service consumer can request management cluster forwarding, but the service provider has final control.
 
 ### User Stories
 
 #### As a service consumer, I want to forward selected API audit logs to my chosen destinations
 
-- configure API audit policy and forwarding using hosted cluster resource.
-- have the desired audit logs forwarded from the management cluster to my destination.
+- Have audit logs forwarded automatically from control-plane CLF to data-plane CLF.
+- Configure API audit policy and forwarding on my data-plane CLF.
+- Forward audit logs in the same way as logs collected from the hosted cluster.
 
 #### As a service provider, I want to forward selected API audit logs to my chosen destinations
+
 - configure the SD internal audit policy
 - configure API audit policy and forwarding using name-spaced logging customer resources in the management cluster.
-  - for example: Red Hat Service Delivery team forwards audit logs to internal splunk destination.
+  - for example: Red Hat Service Delivery team forwards audit logs to internal Splunk destination.
 - independent of user configuration.
 
 ### Goals
 
 - Multiple audit policies can be defined in a management cluster, independent of hosted cluster policies.
 - Multiple audit policies can be defined in a hosted cluster, independent of the management cluster policies.
   See [api-audit-log-policy](/api-audit-log-policy.md) for how the CLO will support this.
-- Allow some forwarding in the _management_ cluster to be configured from the _hosted_ cluster.
-- Keep direct control of management cluster forwarding on the management cluster.
-  - hosted cluster has no direct access to control plane configuration.
-  - data plane resources can request forwarding by the management cluster.
-  - management cluster HLO automatically validates data-plane requests, and enables them if they are safe.
+- Data plane (customer) requests audit log forwarding by modifying data-plane CLF configuration, forwarding from control plane starts automatically.
 
 ### Non-Goals
 
 - This proposal only covers API audit logs. No change to application, infrastructure or node audit logs.
-- Management cluster is not _required_ to honour all possible forwarding configurations allowed by CLF. \
```

Comment:
Will CLF on MC support everything?

Reply:
Yep. I wrote the original constraint because we were using the MC CLF to support both guest and host personas, which was weird, so I wanted to anticipate we might have limitations. Now the guest and host CLF are independent (except for a HTTP forwarding link which isn't a strong coupling) so there's no reason to expect special limitations on either side.